pysat/utils/coords.py

```diff
@@ -147,7 +157,13 @@ def calc_solar_local_time(inst, lon_name=None, slt_name='slt'):
                 slt[i] = hr + inst[lon_name] / 15.0
 
     # Ensure that solar local time falls between 0 and 24 hours
```

Comment:
This comment still needs to be updated.

Reply:
It is fixed over in #727 so I need to make sure this pull doesn't overwrite the other one. I may just copy the updated code from there over here.

Reply:
Checking the commit hashes and dates, these are identical to the first two commits in #727.  Since that branch is approved, I'd merge that in first to see if this goes away before copying any code over.